spec/models/vita_partner_spec.rb

```diff
@@ -32,81 +32,88 @@
     let(:in_range_statuses) { TaxReturnStatus::STATUS_KEYS_INCLUDED_IN_CAPACITY }
 
     context "an organization" do
-      let(:organization) { create :organization, capacity_limit: 10 }
 
       context "at capacity" do
+        let(:at_capacity_org) { create :organization, capacity_limit: 10 }
```

Comment:
When running these tests in parallel, I was running into some issues with sharing the same organization in the top level context -- it seemed like we were getting flaky fails because of it, like maybe the teardown of the other tests weren't finished before we started running the next set of tests.